### PR TITLE
fixed Issue #2131

### PR DESCRIPTION
**Pull request by sdwheeler**
Version(s) of document impacted
------------------------------
- [ ] Impacts 6.1 document
- [ ] Impacts 6.0 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [x] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

File: reference/3.0/Microsoft.PowerShell.Management/Get-HotFix.md [hunk 1]
```diff
@@ -136,10 +136,13 @@ Accept wildcard characters: False
...
+
 ## INPUTS
 
-### None
-You cannot pipe input to Get-HotFix.
+### String
+
+You can pipe one or more computer names to Get-HotFix.
+
 ## OUTPUTS
```
Comment (DCtheGeek) [inline reference/3.0/Microsoft.PowerShell.Management/Get-HotFix.md]: Remove 'Outputs' header to be consistent with changes made to 5/5.1 versions of article.

File: reference/4.0/Microsoft.PowerShell.Management/Get-HotFix.md [hunk 2]
```diff
@@ -146,8 +146,9 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 ## INPUTS
 
-### None
-You cannot pipe input to Get-HotFix.
+### String
+
+You can pipe one or more computer names to Get-HotFix.
 
 ## OUTPUTS
```
Comment (DCtheGeek) [inline reference/4.0/Microsoft.PowerShell.Management/Get-HotFix.md]: Remove 'Outputs' header to be consistent with changes made to 5/5.1 versions of article.

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Management/Get-HotFix.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2131/reference/3.0/Microsoft.PowerShell.Management/Get-HotFix.md) | :white_check_mark:Succeeded | |
[reference/4.0/Microsoft.PowerShell.Management/Get-HotFix.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2131/reference/4.0/Microsoft.PowerShell.Management/Get-HotFix.md) | :white_check_mark:Succeeded | |
[reference/5.0/Microsoft.PowerShell.Management/Get-HotFix.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2131/reference/5.0/Microsoft.PowerShell.Management/Get-HotFix.md) | :white_check_mark:Succeeded | |
[reference/5.1/Microsoft.PowerShell.Management/Get-HotFix.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2131/reference/5.1/Microsoft.PowerShell.Management/Get-HotFix.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C1%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806012018000068-2488%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=5jANDR4Tbor9zLih%2FKYVm1oFhmGXOq2hJmv630Fu6M0%3D&st=2018-06-01T20%3A21%3A52Z&se=2018-07-02T20%3A26%3A52Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/4.0/Microsoft.PowerShell.Management/Get-HotFix.md (hunk 2 above)
Comment (sdwheeler) [reply reference/4.0/Microsoft.PowerShell.Management/Get-HotFix.md]: fixed output section in v5/5.1

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Management/Get-HotFix.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2131/reference/3.0/Microsoft.PowerShell.Management/Get-HotFix.md) | :white_check_mark:Succeeded | |
[reference/4.0/Microsoft.PowerShell.Management/Get-HotFix.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2131/reference/4.0/Microsoft.PowerShell.Management/Get-HotFix.md) | :white_check_mark:Succeeded | |
[reference/5.0/Microsoft.PowerShell.Management/Get-HotFix.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2131/reference/5.0/Microsoft.PowerShell.Management/Get-HotFix.md) | :white_check_mark:Succeeded | |
[reference/5.1/Microsoft.PowerShell.Management/Get-HotFix.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2131/reference/5.1/Microsoft.PowerShell.Management/Get-HotFix.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C1%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806012031379114-2488%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=p8ibNLSXQN1mCTqrECD5JQhgO9vdGdtk2bRKNaW8gJ8%3D&st=2018-06-01T20%3A36%3A16Z&se=2018-07-02T20%3A41%3A16Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**